.gitignore

```diff
@@ -23,6 +23,7 @@ pids
 *.pid
 *.seed
 *.pid.lock
+/examples/trials/benchmarking/automlbenchmark/nni/data
```

Comment:
Please add a `.gitignore` in `/examples/trials/benchmarking/automlbenchmark/nni` and put a data line in it.

Reply:
fixed